docs/sources/features/datasources/cloudwatch.md

```diff
@@ -80,7 +80,7 @@ Here is a minimal policy example:
         "logs:StartQuery",
         "logs:StopQuery",
         "logs:GetQueryResults",
-        "logs:GetLogEvents",
+        "logs:GetLogEvents"
```

Comment:
The comma is common to the other lines, why only remove it from the one line?

Reply:
A trailing comma at the end of the list makes it invalid JSON; it's probably a copy/paste error. Check lines 65-70 in `cloudwatch.md`.